src/reconcile/rec_col.c

```diff
@@ -1071,8 +1073,8 @@ __wt_rec_col_var(
                       (last.start_ts == start_ts && last.start_txn == start_txn &&
                         last.stop_ts == stop_ts && last.stop_txn == stop_txn)) &&
                   ((deleted && last.deleted) ||
-                      (!deleted && !last.deleted && last.value->size == size &&
-                        memcmp(last.value->data, data, size) == 0))) {
+                      (!deleted && !last.deleted && last.value->size != 0 &&
+                        last.value->size == size && memcmp(last.value->data, data, size) == 0))) {
```

Comment:
I don't understand this change, and there's an almost identical bit of code 100 lines earlier in the function that didn't have to change, which concerns me. The commit was `clang sanitiser issuer`, which doesn't help much. Can you please explain what's going on?

Reply:
So there was a clang sanitizer issue flagged with this specific if statement as a result of the changes on this branch. There is a follow up ticket to address this and determine what exactly is triggering the sanitizer.

Reply:
If you have reviewed the code and confirmed that checking `last.value.size != 0` does not change the RLE matching functionality of column-store, it's fbm.

Reply:
Going to review as part of WT-5341.